colors/default.kak

```diff
@@ -17,8 +17,13 @@ face global builtin default+b
 # For markup
 face global title blue
 face global header cyan
-face global bold red
-face global italic yellow
+face global section blue
+face global subsection magenta
+face global subsubsection red
+face global bold red+b
```

Comment:
I do not like those 4 new faces (bold, itallic, underline and strikethrough), we already have attributes to express those (even though strikethrough is not supported pending replacing ncurses).

Regarding section/subsections/subsubsection, we already have header and title which are pretty close, I'd like to see if we can merge those.

Reply:
But `bold`, `italic`, and `underline` are not new faces. I only added `strikethrough` because it was missing and it's good to have a face for it along with those even if it only can be expressed by color.

`Title` face is used to highlight the title, and I think that it is different from `header` (although I'd like to rename `header` to `heading`), because title can be only one per document, and there may be any amount of headings.

Reply:
Oh, yeah, you are right, `bold` and `italic` already exist, however I think I'd rather see them go and just use `default+b` and `default+i`. What we have in the `default` colorscheme is the minimum set of faces a colorscheme must support, and I would like to keep that amount of colors in check.

I think having a single `heading` face shared by all section/subsection/subsubsection is probably enough, eventually we could use `title` for top level sections if we really need a different highlighting for those.

Reply:
Having `bold`, `italic`, `underline`, and `strikethrough` faces is great when you don't have support for those faces in the terminal, and you can code those attributes with colors. Especially with `strikethrough` which isn't supported with `ncurses`. 

In Org Mode, different heading levels represent nodes of a tree structure. Highlighting them in different colors makes semantic meaning in the Org language. And It helps organize TODOs.
Here's an example of agenda in Org mode, whith `(sub){1,2}section` and `header` highlighters using different faces:
![image](https://user-images.githubusercontent.com/19470159/56339155-b418a680-61b5-11e9-9683-136a18476752.png)
Here's how in looks like when all headings use the same face:
![image](https://user-images.githubusercontent.com/19470159/56339209-eaeebc80-61b5-11e9-9add-e270d9eb0571.png)

Emacs screenshots:
![image](https://user-images.githubusercontent.com/19470159/56339501-3a81b800-61b7-11e9-8996-c2138c28d526.png)
Also [concealing](https://github.com/mawww/kakoune/issues/2536) can hide bold, italics, and link markers to make it look more like text.

When [folding](https://github.com/mawww/kakoune/issues/453) will be implemented, I'll add it to this script in order to support this kind of thing:
![image](https://user-images.githubusercontent.com/19470159/56339361-9ef04780-61b6-11e9-9d63-cb518d44a04b.png)
Notice that headings in fzf.kak entry are hidden, as well as text in the four level section.

I think I can move those faces to the Org mode highlighter, and set them by default to default `heading` face. What do you think about it?

Reply:
>  eventually we could use title for top level sections if we really need a different highlighting for those.

`title` has another semantic meaning in Org mode.
Emacs:
![image](https://user-images.githubusercontent.com/19470159/56345870-6b6ae880-61c9-11e9-849b-242574ab126e.png)

Kakoune:
![image](https://user-images.githubusercontent.com/19470159/56345883-74f45080-61c9-11e9-8555-70a2063136ab.png)

Again, I can set all these faces only for Org within the script.